src/DockerContainer.php

```diff
@@ -132,6 +132,10 @@ public function start(): DockerContainerInstance
 
         $process = Process::fromShellCommandline($command);
 
+        // increasing the timeout limit
```

Comment:
Why were these lines added?

Reply:
Sorry, I'll have to open another PR for you, because I'm already using these modifications for my production code.
Let's cancel this PR and I'll open another using another branch. Really sorry about this.